Umbraco-Uno/Working-With-Forms/index.md

```diff
@@ -0,0 +1,28 @@
+---
+versionFrom: 8.0.0
+---
+
+# Working with Forms
+
+## How does Umbraco Forms work
+
+First of all, what is a form? A form is a formula where you can do various things, such as ask support questions. The Umbraco Forms have a wide array of options and can be tailored to fill your exact needs, whether it be a form where the customers can ask support questions or it is a form that asks their preference in pets.
```

Comment:
Perhaps move this below the first heading instead.
That way, it can work as a nice introduction to the entire section.
What do you think?

Also, the sentence is more then 40 words. See if you can split it up, or perhaps shorten it a tiny bit.

Reply:
Interesting ill try it :)